.github/workflows/ci-fuzztest.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON
           -DAVIF_BUILD_TESTS=ON -DAVIF_ENABLE_GTEST=ON -DAVIF_GTEST=LOCAL
           -DAVIF_ENABLE_EXPERIMENTAL_YCGCO_R=ON
-          -DAVIF_ENABLE_EXPERIMENTAL_GAIN_MAP=ON
+          -DAVIF_ENABLE_GAIN_MAP=ON
```

Comment:
Should we delete this (and similar lines in other *.yml files) since this is the default?

Reply:
We can, I left it for now in case we wanted to easily revert the default to OFF. I kept the changes in this PR to the minimum: changing default and renaming the constant. But we can definitely make more changes.